ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/inventory/group_vars/SERVERS.yml

```diff
@@ -45,24 +45,24 @@ servers:
         port_channel:
           structured_config:
             description: Set using structured_config on server adapter port-channel
-          state: present
```

Comment:
did we deprecate state somewhere (or was this from so long ago ? :p )

Reply:
yes, very old! this was just a leftover in molecule. I will make sure all other instances are also cleaned up.